site/docs/components/input/examples.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Input
+layout: DetailComponent
+sidebar:
+ exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+<LivePreview componentName="input" exampleName="Primary" >
+
+### Primary
+
+Input comprises value text and a focus ring. The component is initially empty and then, once it has focus, it accepts text on a single line only.
+
+The default variant for Input is "primary".
+
+- Input is responsive to the size of its parent container, filling it 100%. To disable responsiveness, apply a fixed width and height using the CSS API.
```

Comment:
The positioning of this paragraph may need consideration. This feels only "primary" (given section heading) is doing this